doc/release/0.19.0-notes.rst

```diff
@@ -119,9 +134,18 @@ memory usage.
 The methods `argmin` and `argmax` were added to classes `coo_matrix`,
 `csc_matrix`, `csr_matrix`, and `bsr_matrix`.
 
+New function `scipy.sparse.csgraph.structural_rank` computes the structural
+rank of a graph with a given sparsity pattern.
+
+
 `scipy.special` improvements
 ----------------------------
 
+Scalar, typed versions of universal functions from `scipy.special` are available
+in the Cython space via ``cimport`` from the new module
+`scipy.special.cython_special`. These scalar functions can be expected to be
+significantly faster then the universal functions for scalar arguments.
+
```

Comment:
https://github.com/scipy/scipy/pull/7035/commits/07721efc32e81cf83672501bf40bcae86481efeb refers to the tutorial. Not sure how to hyperlink to it however.